firmware/application/apps/ui_recon.cpp

```diff
@@ -30,14 +30,26 @@ using portapack::memory::map::backup_ram;
 
 namespace ui {
 
+static RecordView* record_view = NULL;
```

Comment:
Maybe should be a class member?

Reply:
I did it so it would not complain about the constructor X-D

Reply:
Your class actually owns the memory in this case so you should use an owning type like suggested below.

Reply:
Add `std::unique_ptr<RecordView> record_view_{};` to your class.

Reply:
I tried and did not succeed. 
Not sure it's what I want in the end, since I need each previous record view to be deleted as it's causing problems if both the audio and raw widgets are loaded at the same time.
Maybe you'll provide us a little PR once this one is merged ? ;-p